server/historian/packages/historian-base/package.json

```diff
@@ -55,12 +55,12 @@
 		"@fluidframework/common-utils": "^1.1.1",
 		"@fluidframework/gitresources": "^3.0.0",
 		"@fluidframework/protocol-definitions": "^1.1.0",
-		"@fluidframework/server-services": "^3.0.0",
-		"@fluidframework/server-services-client": "^3.0.0",
-		"@fluidframework/server-services-core": "^3.0.0",
-		"@fluidframework/server-services-shared": "^3.0.0",
-		"@fluidframework/server-services-telemetry": "^3.0.0",
-		"@fluidframework/server-services-utils": "^3.0.0",
+		"@fluidframework/server-services": "3.1.0-236733",
```

Comment:
Let's use `^3.1.0-236733` for version range

Reply:
Fixed 😄